lib/clockwise_import.rb

```diff
@@ -0,0 +1,211 @@
+require 'csv'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.